packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -87,6 +89,12 @@ export default class ComboBox extends React.Component {
      */
     items: PropTypes.array.isRequired,
 
+    /**
+     * Optional helper function to sort the items in the dropdown list
+     * By default, does not reorder results
+     */
+    dropdownSort: PropTypes.func,
```

Comment:
Maybe `sortItem`, `sortItems`, or `sortByItem` could be a good alternative naming-wise? Just something to suggest that the sort prop relates to the items.